pkg/tecdsa/signing/marshaling.go

```diff
@@ -0,0 +1,91 @@
+package signing
+
+import (
+	"fmt"
+	"github.com/keep-network/keep-core/pkg/crypto/ephemeral"
+	"github.com/keep-network/keep-core/pkg/protocol/group"
+	"github.com/keep-network/keep-core/pkg/tecdsa/signing/gen/pb"
+)
+
+// Marshal converts this ephemeralPublicKeyMessage to a byte array suitable for
+// network communication.
+func (epkm *ephemeralPublicKeyMessage) Marshal() ([]byte, error) {
+	ephemeralPublicKeys, err := marshalPublicKeyMap(epkm.ephemeralPublicKeys)
+	if err != nil {
+		return nil, err
+	}
+
+	return (&pb.EphemeralPublicKeyMessage{
+		SenderID:            uint32(epkm.senderID),
+		EphemeralPublicKeys: ephemeralPublicKeys,
+		SessionID:           epkm.sessionID,
+	}).Marshal()
```

Comment:
I've just realized the ephemeral public key message in the random beacon has a `receiverID` field that is unused. Can we take the opportunity and clean it up?

Reply:
Addressed in https://github.com/keep-network/keep-core/pull/3228/commits/a4e17c9bec28d2bc117652e7a869132d384dc48c